### ISSUE TITLE
Significant Performance Discrepancy in Batch Searches Between @qdrant/js-client-rest and Axios

### ISSUE THREAD

**Issue by csbenjamin** [qdrant/qdrant-js#63]
In batch searches with the `@qdrant/js-client-rest` library, conducting 125 searches with a limit of 30 results each took over half an hour, with Node.js CPU usage hitting 100%. Direct API calls via `axios` completed in under a second, highlighting a significant performance gap.

**Comment by Rendez** [comment]
Could you please share a showcase online?

**Comment by csbenjamin** [comment]
Thank you for your prompt response and for suggesting to share an online showcase to better illustrate the issue I've encountered.

I have attempted to create a standalone demonstration that replicates the problem in a simplified environment. My current setup involves a complex configuration with TypeScript, webpack, and various other libraries. Despite my efforts to isolate the issue, including the use of webpack in a simplified setup, I have been unable to reproduce the performance gap observed in my original environment.

Given these circumstances, I believe it might be best to close this issue for now. The challenge in isolating the problem suggests that it could be related to the specific complexities and interactions within my full project setup, rather than a direct issue with the @qdrant/js-client-rest library itself.

Should I manage to create an isolated example that clearly demonstrates the problem in the future, I will make sure to reopen this case with the new information. I appreciate your understanding and your willingness to assist.

Thank you again for your support.
